src/zimscraperlib/zim.py

```diff
@@ -39,13 +42,23 @@ def __init__(
         self.name = name
         self.tags = tags
         self.scraper = scraper
+        self.source = source
+        self.flavour = flavour
+        self.redirects = redirects
 
     def update(self, **kwargs):
         for key, value in kwargs.items():
             setattr(self, key, value)
 
-    def to_zimwriterfs_args(self):
-        return [
+    def to_zimwriterfs_args(
+        self,
+        verbose=False,
+        inflateHtml=False,
+        uniqueNamespace=False,
+        withoutFTIndex=False,
+        minChunkSize=2048,
```

Comment:
we don't want to set that. If we don't supply a value, we should default to not supply it to zimwriterfs and leave it up to the tool to set it. So use None.

Reply:
I just put 2048 as it is the default value that zimwriterfs uses. Although it shouldn't be a problem but I agree that we should not add this to the argument list unless a value is provided. Will set it to None and add it to arguments only if a value is provided.